### PR TITLE
Fixing drawing of examples

### PR DESCRIPTION
**Pull request by tiago4orion**
The draw wasn't writing at right memory address

### CODE REVIEW

File: _examples/modeset/modeset-double-buffered.go [hunk 1]
```diff
@@ -234,6 +235,9 @@ func draw(file *os.File) {
 		off           uint32
 	)
 
+	//	gcpct := debug.SetGCPercent(-1)
+	//	defer debug.SetGCPercent(gcpct)
```
Comment (vitorarins) [inline _examples/modeset/modeset-double-buffered.go]: I think we can remove these comments.

File: _examples/modeset/modeset.go [hunk 2]
```diff
@@ -212,7 +213,9 @@ func draw() {
 			for k := uint16(0); k < iter.height; k++ {
 				for s := uint16(0); s < iter.width; s++ {
 					off = (iter.stride * uint32(k)) + (uint32(s) * 4)
-					iter.data[off] = (r << 16) | (g << 8) | b
+					val := uint32((uint32(r) << 16) | (uint32(g) << 8) | uint32(b))
+					*(*uint32)(unsafe.Pointer(&iter.data[off])) = val
+					//					iter.data[off] = byte(val)
```
Comment (vitorarins) [inline _examples/modeset/modeset.go]: This one too can be removed.

File: _examples/modeset/modeset-double-buffered.go (hunk 1 above)
Comment (tiago4orion) [inline _examples/modeset/modeset-double-buffered.go]: yes, thanks!
This was just a test to see if the GC was impacting in the drawing performance. But the answer apparently was 'no'.

File: _examples/modeset/modeset.go (hunk 2 above)
Comment (tiago4orion) [inline _examples/modeset/modeset.go]: :+1:

**Pull request merged**